modules/katamari/src/test/ts/atomic/api/str/RepeatTest.ts

```diff
@@ -0,0 +1,26 @@
+import { UnitTest, Assert } from '@ephox/bedrock-client';
+import fc from 'fast-check';
+import * as Strings from 'ephox/katamari/api/Strings';
+
+UnitTest.test('Strings.repeat: unit tests', () => {
+  Assert.eq('Repeat with positive string', '*****', Strings.repeat('*', 5));
+  Assert.eq('Repeat with 0 should be an empty string', '', Strings.repeat(' ', 0));
+  Assert.eq('Repeat with negative should be an empty string', '', Strings.repeat('-', -1));
+});
+
+UnitTest.test('Strings.repeat: positive range', () => {
+  fc.assert(fc.property(fc.char(), fc.integer(1, 100), (s, count) => {
```

Comment:
Try testing it with strings that aren't just single characters also?

Reply:
Done :+1: